.github/gh_matrix_builder.py

```diff
@@ -117,16 +119,18 @@ def build_apache_config(overrides):
 def macos_config(overrides):
     base_config = dict(
         {
-            "pg": PG12_LATEST,
-            "os": "macos-11",
             "cc": "clang",
-            "cxx": "clang++",
-            # the current github macos image has a buggy llvm installation so we build without llvm on mac
-            "pg_extra_args": "--with-libraries=/usr/local/opt/openssl/lib --with-includes=/usr/local/opt/openssl/include --without-llvm",
-            "tsdb_build_args": "-DASSERTIONS=ON -DREQUIRE_ALL_TESTS=ON -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl",
+            "clang": "clang",
             "coverage": False,
-            "installcheck_args": "IGNORES='bgw_db_scheduler bgw_db_scheduler_fixed bgw_launcher pg_dump remote_connection compressed_collation'",
+            "cxx": "clang++",
             "extra_packages": "",
+            "installcheck_args": "IGNORES='bgw_db_scheduler bgw_db_scheduler_fixed bgw_launcher pg_dump remote_connection compressed_collation'",
+            "llvm_config": "/usr/local/opt/llvm/bin/llvm-config",
+            "os": "macos-11",
+            "pg": PG12_LATEST,
```

Comment:
Should we still use PG12 here?

Reply:
I don't know... probably should remove support when we start supporting PG 15.

Reply:
The plan is remove it when we have full PG15 support!